interface/forms/newGroupEncounter/common.php

```diff
@@ -325,7 +325,7 @@ class="css_button link_submit" onClick="top.restoreSession()"><span><?php echo x
      <td class='bold' nowrap><?php echo xlt('Date of Service'); ?>:</td>
      <td class='text' nowrap>
       <input type='text' size='10' class='datepicker' name='form_date' id='form_date' <?php echo $disabled ?>
-       value='<?php echo $viewmode ? substr($result['date'], 0, 10) : date('Y-m-d'); ?>'
+       value='<?php echo $viewmode ? oeFormatShortDate(substr($result['date'], 0, 10)) : oeFormatShortDate(date('Y-m-d')); ?>'
```

Comment:
need to wrap both oeFormatShortDate() with attr() function to prevent xss

Reply:
Done